sdk/src/main/java/com/ingresse/sdk/services/UserService.kt

```diff
@@ -145,4 +159,112 @@ class UserService(private val client: IngresseClient) {
         val type = object : TypeToken<Response<UserUpdatedJSON>>() {}.type
         mUpdateBasicInfosCall?.enqueue(RetrofitCallback(type, callback))
     }
+
+    /**
+     * Get user tickets data
+     *
+     * @param concurrent - parameters to concurrent request
+     * @param request - parameters required to request
+     * @param onSuccess - success callback
+     * @param onError - error callback
+     * @param onConnectionError - connection error callback
+     */
+    fun getUserTicketsData(concurrent: Boolean = false,
+                           request: UserTicketsData = UserTicketsData(),
```

Comment:
I don't think you can send a default UserTicketData() because userId is unique. It must be mandatory.